dockerfiles/ClipperPy35DevDockerfile

```diff
@@ -25,8 +25,7 @@ RUN echo '#!/bin/bash\npython3 "$@"' > /usr/bin/python && \
 
 RUN pip3 install cloudpickle==0.5.* pyzmq==17.0.* requests==2.20.0 scikit-learn==0.19.* \
   numpy==1.14.* pyyaml>=4.2b1 docker==3.1.* kubernetes==5.0.* tensorflow==1.6.* mxnet==1.4.* pyspark==2.3.* \
-  xgboost==0.7.* urllib3==1.24.* # CI is broken when urllib3's version is 1.25.1. Delete urllib3==1.24.* later once version compatibility is stabilized
-
+  xgboost==0.7.* urllib3==1.24.* keras # CI is broken when urllib3's version is 1.25.1. Delete urllib3==1.24.* later once version compatibility is stabilized
```

Comment:
How about set Keras' version like this, `keras==2.2.*`? See https://pypi.org/project/Keras/#history.